test/easyconfigs/easyconfigs.py

```diff
@@ -655,6 +655,8 @@ def check_dep_vars(self, gen, dep, dep_vars):
                 ('2.5.3;', ['medaka-1.5.0-']),
                 # tensorflow-probability version to TF version
                 ('2.8.4;', ['tensorflow-probability-0.16.0-']),
+                # AlphaPulldown
+                ('2.15.1', ['AlphaPulldown-2.0.0b6']),
```

Comment:
This shouldn't be required any longer right?

Reply:
@akesandgren I removed the line
@jkosinski